### PR TITLE
Review list

### PR DESCRIPTION
**Pull request by cxrodgers**
This adds a list view and a detail view for all reviews in the database

### CODE REVIEW

**Comment by cxrodgers** [comment]
As discussed in https://github.com/safwanrahman/jobtalk_beta/issues/4

File: jobtalk/urls.py [hunk 1]
```diff
@@ -16,11 +16,15 @@
...
-from core.views import UserRegistration, successful, ReviewSubmission
+from core.views import (UserRegistration, successful, ReviewSubmission,
+    ReviewList, ReviewDetail)
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^users/', UserRegistration.as_view(), name='user-registration'),
-    url(r'^reviews/', ReviewSubmission.as_view(), name='review-submission'),
+    url(r'^reviews$', ReviewList.as_view(), name='review-list'),
+    url(r'^reviews/$', ReviewList.as_view(), name='review-list'),
```
Comment (safwanrahman) [inline jobtalk/urls.py]: duplicate

File: jobtalk/urls.py [hunk 2]
```diff
@@ -16,11 +16,15 @@
...
+    url(r'^reviews/submit$', ReviewSubmission.as_view(), name='review-submission'),
```
Comment (safwanrahman) [inline jobtalk/urls.py]: missing ``/`` at the end of the url

File: core/models.py [hunk 3]
```diff
@@ -41,3 +42,6 @@ class Review(models.Model):
     travel_allowance = models.IntegerField()
     pension = models.IntegerField()
     quality = models.IntegerField()
+
+    def get_absolute_url(self):
+        return reverse('review-detail', args=[str(self.pk)])
```
Comment (safwanrahman) [inline core/models.py]: only ``self.pk`` should not work?

Comment (cxrodgers) [reply core/models.py]: I was following the django tutorial here: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Generic_views

File: jobtalk/urls.py (hunk 2 above)
Comment (cxrodgers) [reply jobtalk/urls.py]: I was trying to make `jobtalk.com/reviews` and `jobtalk.com/reviews/` go to the same ReviewList view. I don't know enough regex to do it in one line :) Do you think we should remove `jobtalk.com/reviews` from the url list?

Comment (safwanrahman) [reply jobtalk/urls.py]: I think we should follow a url standard. I have seen most of the django dev follows slash at the end of the url. So we should also add `/` at end of the url
I think we can remove the ``/reviews`` from the url list.

File: core/models.py (hunk 3 above)
Comment (safwanrahman) [reply core/models.py]: Oh. I think it should be fixed there also!

File: jobtalk/urls.py (hunk 2 above)
Comment (cxrodgers) [reply jobtalk/urls.py]: @safwanrahman change made

File: core/models.py (hunk 3 above)
Comment (cxrodgers) [reply core/models.py]: I'm not sure exactly how to change it, but "allow edits from maintainers" is checked so feel free..

**Comment by safwanrahman** [comment]
@cxrodgers Merged! 🚀 
Thanks a lot for your contribution!

**Pull request merged**